src/hpc/thread.c

```diff
@@ -103,10 +103,96 @@ void UnlockThreadControl(void)
 #define MAP_ANONYMOUS MAP_ANON
 #endif
 
-#ifndef HAVE_NATIVE_TLS
+#ifndef USE_NATIVE_TLS
+
+#ifdef USE_PTHREAD_TLS
+static int InitTLSKey = 0;
```

Comment:
@rbehrends I honestly don't see how inserting `= 0` "documents that this variable has a default value" (quoted from a previous comment). As I stated back then, all global values default to 0, so this assignment is superfluous. If you indeed want to "document" something here, I would suggest to actually, well, "document" it in the form of a comment that says whatever you want to say.

Reply:
Okay, this is no big deal for me either way. I've changed it.